packages/liferay-npm-scripts/src/utils/run.js

```diff
@@ -19,13 +19,13 @@ function run(executable, ...args) {
 
 	if (signal) {
 		throw new Error(
-			`run(): command \`${command}\` exited due to signal ${signal}`
+			`run(): command \`${command}\` exited due to signal ${signal}.`
 		);
 	}
 
 	if (status) {
 		throw new Error(
-			`run(): command \`${command}\` exited with status ${status}`
+			`run(): command \`${command}\` exited with status ${status}.`
```

Comment:
Here I add the trailing full-stop so that I can do a stricter match, ie. looking for "status 1." as opposed to just "status 1". I could also have used `endsWith()`, but decided to do it this way because it seems good to have a well-defined anchor to match against (for example, if an error ever got dumped in to a log and we wanted to scan inside that, we couldn't use `endsWith()`.